src/main/java/view/InputView.java

```diff
@@ -9,14 +12,25 @@ public class InputView {
     public int getPrice(){
         try{
             System.out.println("구입금액을 입력해 주세요.");
-            int price = scanner.nextInt();
+            int price = Integer.parseInt(scanner.nextLine());
             minCheck(price);
             return price;
         } catch (Exception e){
             throw new IllegalArgumentException("금액은 정수로 입력해주세요.");
         }
     }
 
+    public List<Integer> getAnswer(){
+        try{
+            System.out.println("지난 주 당첨 번호를 입력해 주세요.");
+            List<Integer> answer = Arrays.stream(scanner.nextLine().split(", ")).mapToInt(Integer::parseInt).boxed().toList();
```

Comment:
이렇게 체이닝 걸리는 형태는 메서드마다 한 줄씩 띄어주시는게 가독성 면에서 더 좋을 거 같습니다!
```suggestion
            List<Integer> answer = Arrays
                .stream(scanner.nextLine().split(", "))
                .mapToInt(Integer::parseInt)
                .boxed()
                .toList();
```